underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -110,21 +110,19 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
       UnderFileSystemConfiguration conf) {
 
     String bucketName = UnderFileSystemUtils.getBucketName(uri);
-
-    // Set the aws credential system properties based on Alluxio properties, if they are set
-    if (conf.containsKey(PropertyKey.S3A_ACCESS_KEY)) {
-      System.setProperty(SDKGlobalConfiguration.ACCESS_KEY_SYSTEM_PROPERTY,
-          conf.getValue(PropertyKey.S3A_ACCESS_KEY));
-    }
-    if (conf.containsKey(PropertyKey.S3A_SECRET_KEY)) {
-      System.setProperty(SDKGlobalConfiguration.SECRET_KEY_SYSTEM_PROPERTY,
-          conf.getValue(PropertyKey.S3A_SECRET_KEY));
+    AWSCredentialsProvider credentials;
+    // Set the aws credential system properties based on Alluxio properties, if they are set;
+    // otherwise, use the default credential provider.
+    if (conf.containsKey(PropertyKey.S3A_ACCESS_KEY) &&
+        conf.containsKey(PropertyKey.S3A_SECRET_KEY)) {
+      credentials = new StaticCredentialsProvider(new BasicAWSCredentials(
+          conf.getValue(PropertyKey.S3A_ACCESS_KEY),
+          conf.getValue(PropertyKey.S3A_SECRET_KEY)));
+    } else {
+      // Checks, in order, env variables, system properties, profile file, and instance profile
+      credentials = new DefaultAWSCredentialsProviderChain();
```

Comment:
I think we should only respect this if it is specifically set in the `UnderFileSystemConfiguration`. Currently this will also include if the property is set in the Alluxio base configuration